apps/extension/src/utils/background-script.utils.ts

```diff
@@ -0,0 +1,12 @@
+import { storage } from 'webextension-polyfill';
+
+export const convertLockTime = (value: number) => value * 60 * 1000;
+
+export async function fetchFromStorage<T = any>(key: string): Promise<T | null> {
+  const items = await storage.local.get([key]);
```

Comment:
you are using `storage.local` for storing password hash and session-related data, but you should use  `browser.storage.session`